src/lib/common/globals.h

```diff
@@ -116,7 +116,7 @@
 #define OPT_UNIQUE_VALUES   "unique"
```

Comment:
CNR entry should be added.

In addition, a new section in ngsiv2_implementation_notes.md should be created explaining the existence of this new implementation specific parameter (it some way, it is like "legacy": something that Orion support for performance reasons but not part of the NGSIv2 API spec).